hera_qm/xrfi.py

```diff
@@ -607,11 +685,215 @@ def xrfi_h1c_pipe(uv, Kt=8, Kf=8, sig_init=6., sig_adj=2., px_threshold=0.2,
     else:
         return uvf_f, uvf_wf
 
+
+def xrfi_pipe(uv, alg='detrend_medfilt', Kt=8, Kf=8, xants=[], cal_mode='gain',
```

Comment:
Pipe is a bit of strange name for this, since it's still fairly low level in terms of the entire analysis pipeline. Maybe something like `single_object_xrfi`?

Reply:
It's not a pipeline. It's a single pipe. You can string these together if you'd wish to make something like a pipeline. In this instance it does xrfi, but in previous runs (IDR2.1) it did not. I still like the concept of these functions being "pipes" that you can put together.